db/migrate/20170306042925_associate_affiliate_with_coordinators.rb

```diff
@@ -0,0 +1,6 @@
+class AssociateAffiliateWithCoordinators < ActiveRecord::Migration[5.0]
+  def change
+    add_column :coordinators, :affiliate_id, :integer
+    add_index :coordinators, :affiliate_id
```

Comment:
You have a very good point. But add_index is for improving rails performance of searching in the table. If i didn't add `unique: true`, it will still still allow duplicates.